code/datums/ert.dm

```diff
@@ -71,20 +71,29 @@
 	mission = "Clean up EVERYTHING."
 	polldesc = "a Nanotrasen Janitorial Response Team"
 
-/datum/ert/intern
-	roles = list(/datum/antagonist/ert/intern)
-	leader_role = /datum/antagonist/ert/intern/leader
+/datum/ert/zwei
+	roles = list(/datum/antagonist/ert/zwei)
+	leader_role = /datum/antagonist/ert/zwei/captain
 	teamsize = 7
 	opendoors = FALSE
-	rename_team = "Horde of Interns"
-	mission = "Assist in conflict resolution."
-	polldesc = "an unpaid internship opportunity with Nanotrasen"
+	rename_team = "Zwei Association Raid"
+	mission = "Assist in conflict resolution for Lobotomy Corporation."
+	polldesc = "an opportunity for fixers to make a name for themselves, or die"
```

Comment:
This shows up for ghosts as "Do you want to join as" message, should probably be a proper description of the ERT.

Reply:
Changed the polldescs for the ERT groups.